node/npm/glide/index.ts

```diff
@@ -153,7 +153,7 @@ function initialize() {
         RangeByScore,
         RangeByLex,
         ReadFrom,
-        RedisCredentials,
+        ValkeyCredentials,
```

Comment:
its server credentials in python and java
```suggestion
        ServerCredentials,
```